specutils/io/default_loaders/tabular_fits.py

```diff
@@ -43,3 +45,14 @@ def tabular_fits(file_name, **kwargs):
         data = tab["flux"] * Unit("Jy")
```

Comment:
this is definitely wrong! we could have a separate PR about this, but it is very much unsafe to assume all fluxes are in Jy!

Reply:
@nmearl and @eteq I'd suggest folowing the model of generic_ecsv as a starting point for a generic_tabular_fits and a generic_ipac that call generic_table_from_spectrum to try to figure out which column is which based on the units. I can do this, but probably won't get to it today.